mmv1/products/blockchainnodeengine/go_BlockchainNodes.yaml

```diff
@@ -14,6 +14,7 @@
 # Warning: This is a temporary file, and should not be edited directly
 ---
 name: 'BlockchainNodes'
```

Comment:
uff, this shouldn't be pluralized... but it's been around long enough it probably doesn't make sense to rename :-/